installer/bundle/bundle_skel.sh

```diff
@@ -801,6 +802,16 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+			echo "----- Installing MySQL ruby gem -----"
+			set +e
+			$OMSAGENT_RUBY_DIR/gem install mysql2 2>/dev/null
```

Comment:
This is the omsagent ruby path where we install mysql2 into. This path we build from scratch. Even if there was a different version of even same version of gem install under system/user ruby path, we do not really care :) Our ruby plugins are using the ruby from omsagent path.